tianshou/policy/modelfree/iqn.py

```diff
@@ -0,0 +1,91 @@
+import torch
+import numpy as np
+import torch.nn.functional as F
+from typing import Any, Dict
+
+from tianshou.policy import QRDQNPolicy
+from tianshou.data import Batch
+
+
+class IQNPolicy(QRDQNPolicy):
+    """Implementation of Implicit Quantile Network. arXiv:1806.06923.
+
+    :param torch.nn.Module model: a model following the rules in
+        :class:`~tianshou.policy.BasePolicy`. (s -> logits)
+    :param torch.optim.Optimizer optim: a torch.optim for optimizing the model.
+    :param float discount_factor: in [0, 1].
+    :param int K: the number of samples for policy evaluation. Default to 32.
+    :param int N: the number of samples for online model in training. Default to 8.
+    :param int N_prime: the number of samples for target model in training.
+        Default to 8.
```

Comment:
Do you have other better names for these variables?

Reply:
Good point. Done.